R/cache.R

```diff
@@ -167,16 +167,12 @@ find_new_line <- function(line_number, line, lines) {
 
   while (width <= length(lines)) {
     low <- line_number - width
-    if (low > 0L) {
-      if (lines[low] %==% line) {
-        return(low)
-      }
+    if (low > 0L && lines[low] %==% line) {
```

Comment:
Do you think it is worth assigning this (and also the one below) conditional expression to a Boolean variable with an expressive name?

Reply:
I think the condition is simple enough, and no name jumps out at me at a glance... any suggestion?